go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -437,6 +470,9 @@ var defaultConfig = TabletConfig{
 		IdleTimeoutSeconds: 30 * 60,
 		MaxWaiters:         5000,
 	},
+	Olap: OlapConfig{
+		TxTimeoutSeconds: 0,
+	},
```

Comment:
I think default we should keep to 30 seconds.
I would not recommend `0` for any transaction timeout. It would look the rows for infinite time. 
running a query without a timeout outside a transaction is ok as it does not look any row. But, this goes against the general rule of Vitess that it make things move forward by means of timeouts.

Reply:
OK sounds good to me. As another upside it will mean fewer breaking changes.